fastlane/spec/actions_specs/modify_services_spec.rb

```diff
@@ -15,6 +15,7 @@
           expect(options.associated_domains).to eq('off')
           expect(options.apple_pay).to eq('off')
           expect(options.multipath).to eq('off')
+          expect(options.data_protection).to be_nil
```

Comment:
What does this check for `nil` do here? We already have `expect(options.data_protection).to eq('complete')` on the `:enable` expect. Is this the other side of the same thing?

Reply:
Original implementation is unfortunately written in the way that it is easy possible that services will occur in both lists. After adding tests you have suggested I found another case when it was happening. In following commit I added test cases and fixed the issue.

Reply:
> What does this check for nil do here?

The bug was about that the `data_protection` was enabled, but immediately after that was disabled. In tests there was a false positive - it passed because we expected `data_protection` to be enabled, however we didn't expect (and didn't check) if it was not disabled. This check prevents us from this scenario. 

>  Should all the other options also be nil here and vice versa?

I think that would be a nice improvement 💪

Reply:
Ok, think I got it now. Hard to wrap your around that by just looking at code.

Reply:
Agree, so maybe to be symmetric in the code, we can add the nil check there and vice versa 👍

Reply:
I already added nil checks to all cases in my last commit. Maybe you overlooked it.